temba/channels/models.py

```diff
@@ -146,6 +146,11 @@ def has_attachment_support(self, channel):
         """
         return self.attachment_support
 
+    def get_quick_replies(self, **kwargs):
```

Comment:
Does this need to be on channel type base? Isn't it only ever called from each supporting channel's `send` method?

Reply:
Yeah but isn't it only ever called from the channel type implementation's `send` method? I'm not getting why it needs to be defined on the `ChannelType` base